### ISSUE TITLE
Issue with screen color capture on 4K monitor

### ISSUE THREAD

**Issue by rb0303** [psieg/Lightpack#146]
I have a dual monitor setup, with one 1080p monitor and a second 4K monitor. I installed the LEDs behind the 4K monitor. However, during Prismatik configuration when it asks me to choose between Monitor 1 or Monitor 2, both show up as 1920x1080p resolution. While, I'm able to setup up the capture zones just fine, in actual use, Prismatik only captures from a 1920x1080 area. As a result, only the top left edge of my monitor is mapped correctly to the LEDs. The other edges all capture colors from somewhere around the middle of the screen. 

Is there some way I could make it work with my 4K monitor?

**Comment by Domush** [comment]
I have an identical config and my setup works fine. May be a driver issue.

My config:
nVidia 980ti
1st: 4k monitor
2nd: 1080p monitor
Prismatik monitoring the 4k. Sees it as 4k in the wizard.

**Comment by psieg** [comment]
Indeed, in theory Desktop Duplication is works on any resolution and should easily work with your monitor.

**Comment by rb0303** [comment]
I did find a janky workaround. If I change my 4K panel's resolution to 1080p and then change it back to 4K, then Prismatik works just fine (captures the actual edges of the 4K monitor, instead of from a small section of the monitor). However, upon restarting the computer, the issue comes back. Regardless, both monitors show up as 1080p during the configuration wizard. Do you have any suggestions as to how I might fix this?

**Comment by psieg** [comment]
Are you using windows scaling on the large screen? If you make the capture zones visible, do they show at the screen border or in a top left rectangle?

**Comment by rb0303** [comment]
I am indeed using windows scaling on my 4k monitor (150%). If I make the capture zones visible, they show up normally at the screen border, but they take color input from a top left 1080p sized rectangle only.

**Comment by rb0303** [comment]
Ok, an update - I set windows scaling to 100% and ran the configuration wizard again. Now my 4K monitor shows up 3840x2160, and color is being captured properly at the edges of the screen. So, the issue is definitely due to windows scaling. Unfortunately, it is not feasible for me to continue using my 4K monitor at 100% scaling as text is just too small, and I had to go back to 150%.

**Comment by Domush** [comment]
Try setting the resolution on your 4k to 1440p instead of 4k and see if the problem goes away. I'm running the identical setup as you, but with 175% scaling and resolution of 1440p on my 4k without issue.

Not a solution, but may work as a temp workaround for now.
